<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingItem</class>
 <widget class="QWidget" name="SettingItem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>555</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Droid Arabic Kufi</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>اعدادات المنتجات</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="Jordan"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhExclusiveInputMask</set>
  </property>
  <widget class="QLineEdit" name="lineEditSearchItem">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>اكتوب ما تود البحث عنه</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableSettingItem">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>48</y>
     <width>1231</width>
     <height>501</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>معرف المنتج</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>اسم المنتج</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>صنف المنتج</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>مكان التخزين</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاريخ الاظافة</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاريج الانتاج </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاريخ الانتهاء</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الثمن</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>الكميه</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> المضيف</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnNotActive">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>الغاء المنتج</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboxShowItem">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>المنتجات الفعاله</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>المنتجات الغير فعاله</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnActive">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>تفعيل المنتج</string>
   </property>
  </widget>
  <widget class="QLabel" name="labNumberItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1abc9c;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../pathIcon.qrc"/>
 </resources>
 <connections/>
</ui>
